packages/application-shell/src/components/local-store-provider/local-store-provider.js

```diff
@@ -123,8 +120,10 @@ export const mapStateToProps = (state, ownProps) => ({
 });
 
 export default compose(
-  withRouter, // Used by `withProject`
-  withProject(ownProps => ownProps.match.params.projectKey),
-  withUser(),
+  withApplicationContext(applicationContext => ({
```

Comment:
Doing this to let store be populated with the same test data, which passed to `render*` functions.